### ISSUE TITLE
Accessibility: unable to keyboard navigate just one swiper, when multiple are present

### ISSUE THREAD

**Issue by ajssd** [nolimits4web/swiper#5219]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://jsfiddle.net/asefkow/r2ve7xfs/

### Bug description

For accessibility reasons, we need to enable keyboard navigation on all swipers.  The keyboard module works well for this purpose.  However, when a page contains TWO swipers, then both respond to the same keyboard events.  The user's expectation is that only the "focused" swiper would be respond to the keyboard events.  The jsfiddle demonstrates the issue.

### Expected Behavior

Only the focused swiper should navigate in response to keyboard events

### Actual Behavior

Both swipers navigate in response to keyboard events

### Swiper version

7.3.0

### Platform/Target and Browser Versions

macOS

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by nolimits4web** [comment]
Swiper is kind of isolated/independent thing, one swiper doesn't know anything about other swipers. Use your custom logic to focus/blur what you need and use keyboard methods to enable/disable on required swiper
